vsphere/resource_vsphere_distributed_port_group.go

```diff
@@ -85,17 +92,18 @@ func resourceVSphereDistributedPortGroupRead(d *schema.ResourceData, meta interf
 	if err := validateVirtualCenter(client); err != nil {
 		return err
 	}
-	dvsID := d.Get("distributed_virtual_switch_uuid").(string)
 	pgID := d.Id()
-	pg, err := dvPortgroupFromUUID(client, dvsID, pgID)
+	pg, err := dvPortgroupFromMOID(client, pgID)
```

Comment:
The method signature of `dvPortgroupFromMOID` changes here, but I don't see that in the changes here. Was it changed elsewhere, and this resource currently doesn't work without this change?

Reply:
Not just the signature, but the method as well :slightly_smiling_face:

`dvPortgroupFromUUID` vs `dvPortgroupFromMOID`

Reply:
![ohisee](https://user-images.githubusercontent.com/61721/31516470-461b6ee8-af5e-11e7-9c9b-e5fae2d9a11b.gif)